### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Access Keys_Keyboard:Focus is not visible on ribbon bar controls.

### ISSUE THREAD

**Issue by Accessibility56** [Microsoft/IIS.WebManager#79]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #.NETWebsites; #Keyboard; #WCAG2.4.7; #C+E-H; #Chrome

**Environment Details:**
**URL**: https://localhost:55539/security/tokens
**Browser Details:** Chrome(68.0.3440.106)
Keyboard

**Repro Steps:**
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till "Skip This" button and Press Enter.
4.Tab till "Get Access Token" link and Press Enter.
5.Tab to all controls on page and verify keyboard focus is visible on controls of page.

**Actual Result:** 
Focus is not visible on ribbon bar controls.

**Expected Result:**
Keyboard focus should be visible on each and every control of page.

**User Impact:**
Keyboards users may not know where focus is if keyboard focus is not visible on few controls of page. They can not use the application properly.

**Attachment:**
![bug_79_access keys_focus visible](https://user-images.githubusercontent.com/38523250/44581766-ac3a6b00-a7bc-11e8-9fdc-241c0be5cc59.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
